### ISSUE TITLE
New minor release

### ISSUE THREAD

**Issue by fridex** [thoth-station/thamos#202]
Depends-On: https://github.com/thoth-station/thamos/pull/191
Depends-On: #203

**Comment by harshad16** [comment]
ah too soon, needed some additional files

**Comment by harshad16** [comment]
i need to raise a additional PR by running swagger_codegen on user-api latest

**Comment by fridex** [comment]
> i need to raise a additional PR by running swagger_codegen on user-api latest

Ah, right. The release has been postponed.

**Comment by fridex** [comment]
Dropping WIP flag.
